go.mod

```diff
@@ -3,7 +3,7 @@ module github.com/edgexfoundry/app-functions-sdk-go
 require (
 	github.com/BurntSushi/toml v0.3.1
 	github.com/edgexfoundry/go-mod-core-contracts v0.0.0-20190213225129-211d234d86e4
-	github.com/edgexfoundry/go-mod-registry v0.0.0
+	github.com/edgexfoundry/go-mod-registry v0.0.0-20190221211212-171439dc16f8
```

Comment:
Remove the `replace` directive at the bottom of this file

Reply:
Yeah, this is what likely caused the build failure